pkg/kubelet/dockertools/manager.go

```diff
@@ -276,6 +276,38 @@ type containerStatusResult struct {
 	err    error
 }
 
+// determineInfraContainerIP inspects the given container and determines the IP address of that
+// container.  The container is assumed to be the pod infra container; it is the responsibility
+// of the caller to ensure that the container ID passed is that of the infra container.
+func (dm *DockerManager) determineInfraContainerIP(dockerID string, pod *api.Pod) (string, error) {
+	inspectResult, err := dm.client.InspectContainer(dockerID)
+	if err != nil {
+		return "", err
+	}
+	if inspectResult == nil {
+		return "", nil // TODO: should be an error, probably
```

Comment:
Make these errors, then the caller can choose to ignore the error.  If you can't get a network IP it's better to continue with no status IP.  Don't log in this method.